src/Custom/Responses/StreamingResponseCompletedUpdate.cs

```diff
@@ -0,0 +1,22 @@
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenAI.Responses
+{
+    public partial class StreamingResponseCompletedUpdate : StreamingResponseUpdate
```

Comment:
Looking at all the customizations for the Streaming types in this PR, it appears that these were customized to change the use of `OpenAIResponse` to use `ResponseResult` instead. This made sense when these were two separate types. However, now `ResponseResult` has fully replaced `OpenAIResponse`, and as such, we should be able to remove this dozen of custom files and let it be generated instead. 🙂